### ISSUE TITLE
BUG/QUESTION: Unpin behavior 

### ISSUE THREAD

**Issue by bkraul** [RocketChat/Rocket.Chat#14452]
### Description:

I accidentally pinned one of my messages during a conversation. This caused the message to be posted with a quote style. When I tried to unpin, nothing happened. I was expecting the "pinned" message to disappear.

### Steps to reproduce:

1. Download latest docker develop build (4ba3513).
2. Type a message.
3. Pin it. Will see the message being pinned.
4. Attempt to unpin the message. Nothing happens. Pinned message remains.

### Expected behavior:
I would expect the pinned/quoted message to be removed when clicking Unpin on the original message, as the pinned message itself has no option to remove.

### Actual behavior:

Pinned message remains. See GIF.

![unpin](https://user-images.githubusercontent.com/14616851/57524322-18c6ad00-72ed-11e9-838e-181fb05ca6bb.gif)

### Rocket.Chat

|                 Version | 1.1.0-develop                                             |
| ----------------------: | --------------------------------------------------------- |
|     Apps Engine Version | 1.4.2                                                     |
|      Database Migration | 144                                                       |
| Database Migration Date | Fri May 10 2019 06:18:56 GMT-0500 (Central Daylight Time) |
|            Installed at | Fri Jan 25 2019 17:44:08 GMT-0600 (Central Standard Time) |
|                  Uptime | 1 minutes, 25 seconds                                     |
|           Deployment ID | dwPkHnPwaKuHALm6b                                         |
|                     PID | 1                                                         |
|       Running Instances | 1                                                         |
|                   OpLog | Enabled                                                   |

### Commit

|    Hash | 4ba35136153033a021b75fb3c51d40976f8674c5 |
| ------: | ---------------------------------------- |
|    Date | Thu May 9 18:08:17 2019 -0300            |
|  Branch | develop                                  |
|     Tag | 1.0.1                                    |
|  Author | Guilherme Gazzo                          |
| Subject | fix (#14443)                             |

**Comment by MarcosSpessatto** [comment]
@bkraul actually this is the expected behavior, when you pin a message, an announcement is inserted in the channel (like you show on your GIF). When you unpin the message, the message is removed from the list of pinned messages, but the announcement keeps in the channel (like an history of actions). See https://github.com/RocketChat/Rocket.Chat/issues/7124#issuecomment-315409710

**Comment by bkraul** [comment]
Not sure I follow. If I unpin the message, the announcement should be unpinned, because as I mentioned, an "Unpin" action seems to do absolutely nothing. Take for instance what happened to me. I was trying to edit the message. But I accidentally pinned it. I quickly unpinned it, but the announcement stayed behind, causing confusion with the people I was communicating with. This is more apparent on DM channels.

**Comment by bkraul** [comment]
At least adding a delete option for the actual announcement would go a long way. The problem is that it cannot be deleted by any means.

**Comment by ChristineBoersen** [comment]
I had to disable it currently since it was confusing too many of my users. Being able to delete the left behind pinning would really help.

**Comment by LeeThompson** [comment]
This happens to me too, pin/unpin seems largely useless.

**Comment by matacino** [comment]
Yes, it is also for useless and irritating for our users, that an accidentally "pinned" message does not disappear whe unpinned

**Comment by audouts** [comment]
I have an annoying announcement that is left over from a deleted message. Is there a way to delete it, too?

I understand that pin/unpin announcements are part of channel history but they should not persist for deleted messages.

I also agree that it would be better to make them optional.
